tests/test_qa_a3_seed_data.py

```
@@ -0,0 +1,120 @@
+    # 开局清洗后进人才池的两位，必须在 seed office 串里带罢居标记
+    assert {"袁崇焕", "孙承宗"} <= bajiu_names
+
+    stage = _liaodong_seed().stage_text
+    for name in ("袁崇焕", "孙承宗"):
+        assert name not in stage, f"辽东索饷 stage_text 仍点名罢居者 {name!r}: {stage!r}"
+
+    # 若点到名册人物，每人须非罢居串；允许匿名将领/边情质感
+    candidates = set(re.findall(r"[\u4e00-\u9fff]{2,4}", stage))
+    named = candidates & set(characters)
+    assert named or ("将" in stage or "边" in stage), (
+        "stage_text 既未点名册人物，也未保留将领/边情叙事质感"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)